www/js/main.js

```diff
@@ -96,8 +96,29 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
     }
 })
 
-.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope) {
+.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope, ControlHelper) {
+    $scope.emailLog = ControlHelper.emailLog;
     $scope.dark_theme = $rootScope.dark_theme;
+
+    
+    $scope.toggleLowAccuracy = function() {
+        if ($scope.settings.collect.config == null) {
+            return false; // config not loaded when loading ui, set default as false
```

Comment:
I am not sure that this is the right thing to do. This means that the low accuracy will return `false` even the user set it before. This is going to cause a lot of confusion - see bugs like https://github.com/e-mission/e-mission-phone/issues/53

Isn't the profile screen basically the rebranded control screen? The control screen loads all the settings on startup. If it doesn't right now, it should. Otherwise, we will get bugs in which it will look like the user is not logged in although they are and so on.